src/components/QuestionerButton/QuestionerButton.css

```diff
@@ -12,18 +12,26 @@
 .next {
   min-width: 203px;
   padding: 20px;
-  background: rgba(55, 31, 9, 0.6);
-  border: 4px solid rgba(55, 31, 9, 0.7);
+  background: rgba(55, 31, 9, 60%);
+  border: 4px solid rgba(55, 31, 9, 70%);
   border-radius: 2px;
   color: #f1f1e6;
 }
+.questioner-button .back:active {
+  background: #f1f1e6 !important;
+  border: 4px solid rgba(55, 31, 9, 80%);
+}
+.questioner-button .next:active {
+  background: #f1f1e6 !important;
+  border: 4px solid rgba(55, 31, 9, 80%);
+}
 
 .questioner-button,
 .back:hover {
-  background-color: #170c01;
+  background-color: #371709;
 }
 
 .questioner-button,
 .next:hover {
-  background-color: #170c01;
+  background-color: #371709;
 }
```

Comment:
Since the styles are the same for `.back` and `.next`, it makes no sense to repeat them.

Just go with selectors `.questioner-button:hover` and `.questioner-button:active`, there is no need for `.back` and `..next` at all.

Additionally please avoid `!important`.

Reply:
I'll add the correct colors for you in the `index.css` file as CSS variables, let's just use those.